Content.Shared/MagicMirror/SharedMagicMirrorSystem.cs

```diff
@@ -42,7 +42,7 @@ private void OnMirrorRangeCheck(EntityUid uid, MagicMirrorComponent component, r
             return;
         }
 
-        if (!_interaction.InRangeUnobstructed(uid, component.Target.Value))
+        if (!_interaction.InRangeUnobstructed(uid, component.Target.Value, SharedInteractionSystem.InteractionRange, Physics.CollisionGroup.None))
```

Comment:
Wallmounts usually just handle this shit on its own? I wonder if you change the check from mirror -> target to target -> mirror if the wallmount code will handle it properly.

Reply:
yeah, I deleted it so it would handle it on its own
then you mentioned the scissors use the exact same code, so I'm basically just implementing code that would've been automatically checked on the wallmount stuff just for the sake of the scissors also working